src/main/java/com/uipath/uipathpackage/models/TestOptions.java

```diff
@@ -73,4 +78,45 @@ public boolean getAttachRobotLogs() {
 	public void setAttachRobotLogs(boolean attachRobotLogs) {
 		this.attachRobotLogs = attachRobotLogs;
 	}
+
+    public String getRepositoryUrl() {
```

Comment:
Same question as for the PackOptions new getters.

Reply:
same as splitOutput